db/migrate/20151013135415_add_pub_key_to_smart_proxy.rb

```diff
@@ -0,0 +1,5 @@
+class AddPubKeyToSmartProxy < ActiveRecord::Migration
+  def change
+    add_column :smart_proxies, :pubkey, :text
```

Comment:
does it make sense to reuse the existing KeyPair model instead? (would need probably re factoring but it all stays in one place)